jax/_src/numpy/lax_numpy.py

```diff
@@ -3555,8 +3571,25 @@ def fromstring(string: str, dtype: DTypeLike = float, count: int = -1, *, sep: s
   return asarray(np.fromstring(string=string, dtype=dtype, count=count, sep=sep))
 
 
-@util.implements(np.eye)
+@util.implements(np.eye, extra_params="""
+device : :py:class:`Device`, :py:class:`Sharding`, optional
+  The (optional) :py:class:`Device`, :py:class:`Sharding`,
+  representing the device(s) to which created array should be
+  transferred. If given, then the result is committed to the device(s).
+""")
 def eye(N: DimSize, M: DimSize | None = None,
```

Comment:
Changes to `eye`, `linspace`, `arange`, etc. are not mentioned in the PR description. Perhaps this is a stray commit?

Please update the PR with the portions you want to be reviewed here, and let me know. Thanks!